electricitymap/contrib/capacity_parsers/CL.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
```

Comment:
Should this file be renamed to only cover the specific subzone (`CL-SEN`)? Otherwise I think we should make it clear that this data is not valid for the other subzones :)